app/assets/stylesheets/application.css.scss

```diff
@@ -5,3 +5,8 @@
 @import 'identity-style-guide/dist/assets/scss/styles';
 @import 'components/all';
 @import 'print';
+
+
+body {
+  background-image: asset-url('dne.png');
+}
```

Comment:
Looks like this snuck in — might need one more commit (and maybe then a squash of these, if you want to tidy up the history before merging).

Reply:
Ah yep, this snuck in too. I'll put up uno mas and squash right before merging